### PR TITLE
PSP-4043 fix

### PR DESCRIPTION
**Pull request by asanchezr**
Screenshot 1

------------

Screenshot 2

### CODE REVIEW

File: frontend/src/features/properties/map/research/ResearchTabsContainer.tsx [hunk 1]
```diff
@@ -8,9 +8,16 @@ import { Api_ResearchFile } from 'models/api/ResearchFile';
...
 
 export interface IResearchTabsContainerProps {
   researchFile: Api_ResearchFile;
+  // The "edit key" identifies which form is currently being edited: e.g.
+  //  - property details info,
+  //  - research summary,
+  //  - research property info
+  //  - 'none' means no form is being edited.
+  setEditKey: (editKey: FormKeys) => void;
   setEditMode: (isEditing: boolean) => void;
```
Comment (asanchezr) [inline frontend/src/features/properties/map/research/ResearchTabsContainer.tsx:21]: I had to add one more indicator/flag to make sure the appropriate update sub-form was shown given the various "edit" buttons for different tabs/sections

File: frontend/src/features/properties/map/research/ViewSelector.tsx [hunk 2]
```diff
@@ -38,18 +52,29 @@ const ViewSelector: React.FunctionComponent<IViewSelectorProps> = props => {
...
+        );
+      } else {
+        return (
+          <UpdatePropertyView
+            researchFileProperty={researchFileProperty}
+            setFormikRef={props.setFormikRef}
+            onSuccess={props.onSuccess}
+          />
+        );
+      }
```
Comment (asanchezr) [inline frontend/src/features/properties/map/research/ViewSelector.tsx:71]: The logic on ViewSelector is becoming hard to follow with multiple nested if statements. This would be a good candidate for refactoring but will not do as part of the bug fix - should be addressed separately (in my opinion)

**Review by devinleighsmith — APPROVED** [review]
